lib/status-routes.schema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "hide": true,
+  "response": {
+    "200": {
+      "type": "object",
+      "required": ["status"],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the current service that has responded"
+        },
+        "status": {
+          "type": "string",
+          "description": "It\"s status, it must be OK",
+          "enum": ["OK"]
+        }
+      }
+    },
+    "503": {
+      "type": "object",
+      "required": ["status"],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the current service that has responded"
+        },
+        "status": {
+          "type": "string",
+          "description": "It's status, it can be KO, or a human readable reason for failure"
```

Comment:
Why KO? I suggest just human readable

Reply:
It’s a sensible default if the probe is operated only for things like docker or kubernetes where no data from the call is surfaced, is useless to write a pretty string that no one ever will read.

Obviously we will encourage the humane readable one every time it will make sense.